projects/aca-content/src/lib/components/knowledge-retrieval/search-ai/agents-button/agents-button.component.html

```diff
@@ -26,8 +26,10 @@
         [attr.data-automation-id]="'aca-agents-button-agent-' + agent.id"
         [value]="agent">
         <div class="aca-agents-button-menu-list-agent-content">
-          <adf-avatar [initials]="initialsByAgentId[agent.id]"></adf-avatar>
-          {{ agent.name }}
+          <adf-avatar src="{{ agent?.avatar }}" [initials]="initialsByAgentId[agent.id]"></adf-avatar>
```

Comment:
In most cases we do [] instead of {} for inputs so can you please change it to 
[src]="agent?.avatar" ?